webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpacesCardList.vue

```diff
@@ -157,8 +157,7 @@ export default {
     },
     searchSpaces() {
       this.loadingSpaces = true;
-      this.loadingSpaces = true;
-      return this.$spaceService.getSpaces(this.keyword, this.offset, this.limitToFetch, this.filter)
+      return this.$spaceService.getSpaces(this.keyword, this.offset, this.limitToFetch, this.filter, 'favorite')
```

Comment:
Please add this expand information for Search card as well ( here: https://github.com/Meeds-io/social/blob/761c6d437aa8219addece23c43cec74688ad8b1f/extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/search-configuration.xml#L97 )